src/renderer/coremods/contextMenu/plaintextPatches.ts

```diff
@@ -7,7 +7,10 @@ export default [
       {
         match: /var \w,\w=(.)\.navId/,
         replace: (vars, menu) =>
-          `replugged.coremods.coremods.contextMenu._insertMenuItems(${menu});${vars}`,
+          `if (!${menu}.plugged){` +
```

Comment:
Do these checks in the function itself.

Reply:
okay first check if statement is to check if it's remade menu which can't be done inside function or else it will be infinite loop kinda. second if statement is there to check if we returned null/undefined and that's only if we can't get the this function from module itself so this can't be inside function either.
if you know how this can be inside function then I am open, but to my knowledge it needs to be handled here.